### PR TITLE
refactor: radii and keyframe

### LINKED ISSUE #4637: Update corner radius scale and components
**New corner radius scale:**
- XS: 2px – buttons, rows, collectibles images;
- SM: 4px – menus, drop-down, inputs;
- MD: 8px – modals and drawers;
- LG: 12px – page container;

**Task:**

- [x] Create new scale for corner radius
- [ ] Update components

------

FINAL PROPOSAL: OPTION 1
https://www.figma.com/file/2MLHeIeL6XPVi3Tc2DfFCr/%E2%9D%96-Leather-%E2%80%93-Design-System?type=design&node-id=9972-108739&mode=design&t=oU8qhxS
border radius

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of Leather — [Extension build](https://github.com/leather-wallet/extension/actions/runs/7530343353), [Test report](https://leather-wallet.github.io/playwright-reports/refactor/radii-and-keyframe)

This PR updates components with the new radii token sizes. I'm not sure it is possible to catch all design changes here, but I did my best and def a good step. I think the container work should rebase on these changes once merged and do another pass to make sure matches new designs. cc @pete-watters

### CODE REVIEW

File: src/app/components/secret-key/mnemonic-key/mnemonic-input-field.tsx [hunk 1]
```diff
@@ -69,7 +70,7 @@ export function InputField({ dataTestId, name, onPaste, onChange, value }: Input
     >
       <TextField.Slot className={css({ padding: 'space.00', marginRight: 'space.01' })}>
         {/* // FIXME #4130: - update this color when available in design system */}
-        <styled.span textStyle="label.01" color="GrayText">{`${name}.`}</styled.span>
+        <styled.span textStyle="label.01" color="accent.text-subdued">{`${name}.`}</styled.span>
```
Comment (pete-watters) [inline src/app/components/secret-key/mnemonic-key/mnemonic-input-field.tsx:72]: I think we can ditch the `FIXME #4130` above now 👍

File: src/app/components/secret-key/mnemonic-key/mnemonic-input-field.tsx [hunk 2]
```diff
@@ -69,7 +70,7 @@ export function InputField({ dataTestId, name, onPaste, onChange, value }: Input
     >
       <TextField.Slot className={css({ padding: 'space.00', marginRight: 'space.01' })}>
         {/* // FIXME #4130: - update this color when available in design system */}
```
Comment (pete-watters) [inline src/app/components/secret-key/mnemonic-key/mnemonic-input-field.tsx]:
```suggestion
```

File: src/app/components/pill.tsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ export function Pill({ hoverLabel, label, ...props }: PillProps) {
     <Tooltip disabled={!hoverLabel} label={hoverLabel} maxWidth="200px" placement="bottom">
       <Box
         border="1px solid"
```
Comment (pete-watters) [inline src/app/components/pill.tsx]: Maybe we can update the border also? 

```suggestion
        border="default"
```

**Review by pete-watters — APPROVED** [review]
Great work here @fbwoolf and thanks for the heads up. 

Code LGTM  👍

**Review by kyranjamie — APPROVED** [review]

**Pull request merged**